Module Added

sourav8256 opened the pull request.

Sashi20 commented:
Give proper commit messages when you push code. Do not use messages like "First commit" and so on.

sourav8256 committed:
1. New banner creation form
2. Interface to see all banners in groups (Recent/Active/Inactive)
3. Interface to select the sites in which banner will be displayed and delete banners
4. Interface to set banners active/inactive
5. Interface to add new websites to the list
6. Block for fetching and displaying banners on different websites
7. Form to set variables (in settings.inc file)
